### PR TITLE
add tools

### PR DESCRIPTION
**Pull request by bxue-l2**
## Checks

- [ ] I've made sure the lint is passing in this PR.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, in that case, please comment that they are not relevant.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Integration tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by bxue-l2** [comment]
@anupsv can you approve this PR?

**Comment by anupsv** [comment]
The tool verified correctly. Below are the details:
```
-rw-rw-r-- 1 ubuntu ubuntu 8.0G Mar 29 04:55 g1.point
-rw-rw-r-- 1 ubuntu ubuntu  16G Mar 29 04:55 g2.point
-rw-rw-r--  1 ubuntu ubuntu  97G Jan  1 23:15 challenge_0085
ubuntu@ip-172-31-5-7:~/eigenda/tools/srs-utils$ sha256sum g1.point
c03178df1bafd883dee748205dae7725bcf58cead10f6bb1b07aeb51e5aede6f  g1.point
ubuntu@ip-172-31-5-7:~/eigenda/tools/srs-utils$ sha256sum g2.point
e3927cccc1bc9ace5a41e950cd50cb9e1c3fc843975f20130e6eb294414a3c50  g2.point

In total, we will verify 5000 batches. Each batch contains 53688 points.
For the first 3 batches, we show the time taken to verify each batch, then estimate the total verification hours.
After the first 3 batches, we will update every 250 batches
Verify 1 batch takes 4.661458011s. Verify takes 3.823695611s
verify 5000 batches will take 5 Hours
Verify 1 batch takes 4.76100127s. Verify takes 3.875402219s
verify 5000 batches will take 5 Hours
Verify 1 batch takes 4.707741451s. Verify takes 3.829696617s
verify 5000 batches will take 5 Hours
Verified 250-th batches. Time spent so far is 19m46.896060959s
Verified 500-th batches. Time spent so far is 39m34.812989621s
Verified 750-th batches. Time spent so far is 59m17.396713225s
Verified 1000-th batches. Time spent so far is 1h18m59.684984393s
Verified 1250-th batches. Time spent so far is 1h38m41.334846118s
Verified 1500-th batches. Time spent so far is 1h58m20.049931996s
Verified 1750-th batches. Time spent so far is 2h17m59.007347849s
Verified 2000-th batches. Time spent so far is 2h37m36.799471289s
Verified 2250-th batches. Time spent so far is 2h57m13.524680496s
Verified 2500-th batches. Time spent so far is 3h16m49.243964321s
Verified 2750-th batches. Time spent so far is 3h36m23.103187031s
Verified 3000-th batches. Time spent so far is 3h55m57.633739864s
Verified 3250-th batches. Time spent so far is 4h15m33.372590255s
Verified 3500-th batches. Time spent so far is 4h35m9.03037648s
Verified 3750-th batches. Time spent so far is 4h54m44.313370148s
Verified 4000-th batches. Time spent so far is 5h14m20.279767628s
Verified 4250-th batches. Time spent so far is 5h33m55.762074186s
Verified 4500-th batches. Time spent so far is 5h53m31.005833781s
Verified 4750-th batches. Time spent so far is 6h13m6.217374283s
Done. Everything is correct

```

**Review by anupsv — APPROVED** [review]

File: tools/srs-utils/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+It produces two files, g1.point and g2.point. g1.point contains 8589934592 Bytes and g2.point 17179869184 Bytes
+
+This procedure takes roughly 10 minutes.
+
+Note: The challenge files contains 2^29 G1 points and 2^28 G2 points with secret tau. We use only the first 2^28 G1 points for EigenDA.
+
+### How to verify the parsed G1, G2 points
+
+`go run main.go parse --g1-path <Path to g1.point> --g2-path <Path to g2.point>`
```
Comment (anupsv) [inline tools/srs-utils/README.md:34]: This needs to be `verify` not `parse`

**Review by jianoaix — APPROVED** [review]

File: tools/srs-utils/README.md (hunk 1 above)
Comment (bxue-l2) [reply tools/srs-utils/README.md:34]: good catch

**Pull request merged**